.gitignore

```diff
@@ -1,4 +1,11 @@
 target
+target_0
```

Comment:
I'd do some kind of wildcard already here

Reply:
Done